calendar.rb

```diff
@@ -0,0 +1,98 @@
+require './event.rb'
+class Calendar
+  def initialize
+    @events = {}
+  end  
+
+  def add_event
+    while true
+      print "Enter the name of Event: "
+      name = gets.chomp
+      if !name.empty?
+        break
+      end
+    end
+    print "Enter the details of Event: "
+    detail = gets.chomp
+    print "Enter the venue of Event: "
+    venue  = gets.chomp
+    puts "Enter date"
+    date   = get_date
+    event  = Event.new(name, detail, venue, date)
+    if @events.has_key?(date)
+      @events[date] << event
+    else 
+      event_day = Array.new
```

Comment:
variable name suggests that it will contain a day but it contains an array of events. so the name can be `day_events`.